include/pocl_file_util.h

```diff
@@ -48,8 +48,6 @@ int pocl_rename(const char *oldpath, const char *newpath);
 
 int pocl_exists(const char* path);
 
-int pocl_filesize(const char* path, uint64_t* res);
```

Comment:
Why do you remove this function? If it's not called at all now, maybe we can keep it around for later needs for some time?